src/jsdoc.ts

```diff
@@ -37,44 +37,66 @@ export interface Tag {
  * The public Closure docs don't list all the tags it allows; this list comes
  * from the compiler source itself.
  * https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/parsing/Annotation.java
+ * https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/parsing/ParserConfig.properties
```

Comment:
This seems a bit risky -- it means we pass through all these comments verbatim.  Is that your intent?